browser/sync_usb.js

```diff
@@ -41,33 +44,29 @@ pump(ws, split(JSON.parse), through.obj(function (row, enc, next) {
     if (replicationProgress === 3) resdiv.innerHTML += '|'
   }
   next()
-})).on('error', onerror)
-
-function onerror (err) { log.error(err) }
+})).on('error', function (err) { log.error(err) })
 
 var resdiv = document.getElementById('response')
 var cancelBtn = document.getElementById('cancel')
 var selectExistingBtn = document.getElementById('select-existing')
 var selectNewBtn = document.getElementById('select-new')
 var sourceField = document.querySelector('form#sync input[name="source"]')
 
-ipc.on('select-file', function (event, file) {
-  if (!file) return
-  sourceField.value = file
```

Comment:
I think this was to show to the user the file they had selected? Do you think we should remove this?

Reply:
It seems like that input box has been hidden in a previous PR. I agree it'd be a good idea to show the user the file they had selected, though.